CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Data migrations
 - Rename filesystem directory containing workspaces from `users` to `workspaces`
 ### Changes
+- [#159] (https://github.com/jirik/layman/issues/159) Object `layman_metadata` was added to [GET Workspace Layer](doc/rest.md#get-workspace-layer) and [PATCH Workspace Map](doc/rest.md#patch-workspace-map) responses.
```

Comment:
Attribute `layman_metadata.publication_status` can be used for watching global state of publication (updating, complete, incomplete).